config/jobs/ppc64le-cloud/docker-in-docker/postsubmit-push-docker-in-docker.yaml

```diff
@@ -0,0 +1,98 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
```

Comment:
@florencepascual Was this job tested on a ppc64le k8s cluster?

Reply:
yes

Reply:
I could not, since I don't have the quay-powercloud-regcred. 
I got inspiration from https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml and tested all I could in a pod.

Reply:
@florencepascual In cases where we dont have quay-powercloud-regcred, We usually test it with other quay credentials before submitting the PR.
Do you have a x86 and ppc cluster to test this job completely?

Reply:
We have the clusters, I will try to test it.